contracts/test/unit/governance/NopeDAO.t.sol

```diff
@@ -77,7 +77,7 @@ contract NopeDAOTest is DSTest {
         assertEq(votingDelay, 0);
 
         uint256 votingPeriod = nopeDAO.votingPeriod();
-        uint256 fourDays = 86400 * 4;
+        uint256 fourDays = 26585;  // (86400 * 4) / 13
```

Comment:
OZ governor has this parameter in units of blocks rather than seconds. Make a mistake in the original config, currently reports 22 months